library/src/main/java/com/datatorrent/lib/stream/StreamMerger.java

```diff
@@ -44,6 +45,7 @@
   /**
    * Data input port 1.
    */
+  @InputPortFieldAnnotation(optional = true)
```

Comment:
This won't be needed if we don't use mergers with just one input. See comment above on number of mergers.